sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsAsyncClientTests.java

```diff
@@ -459,6 +503,52 @@ public void updateRoomWithInvalidRoomId(HttpClient httpClient) {
                 }).verifyComplete();
     }
 
+    @ParameterizedTest
+    @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    public void updateRoomWithNoParameters(HttpClient httpClient) {
+        roomsAsyncClient = setupAsyncClient(httpClient,
+                "updateRoomWithNoParameters");
+        assertNotNull(roomsAsyncClient);
+
+        CreateRoomOptions createRoomOptions = new CreateRoomOptions()
+                .setValidFrom(VALID_FROM)
+                .setValidUntil(VALID_UNTIL)
+                .setPstnDialOutEnabled(true);
+
+        Mono<Response<CommunicationRoom>> response1 = roomsAsyncClient.createRoomWithResponse(createRoomOptions);
+
+        StepVerifier.create(response1)
+                .assertNext(roomResult -> {
+                    assertHappyPath(roomResult, 201);
+                })
+                .verifyComplete();
+
+        String roomId = response1.block().getValue().getRoomId();
```

Comment:
Async tests should ideally not use `block()`. Use `StepVerifier`.

Reply:
I see block being used everywhere in this file to get Room id, We are uisng StopVerifier but to get RoomId we are using block(). Is there any other way we can achieve this?